src/main/java/productDependency/CreateOrderPage.java

```diff
@@ -0,0 +1,1481 @@
+		CreateOrderPage sp = new CreateOrderPage(driver);
+		log.info("Verifying the login button is available or not");
+		Assert.assertTrue(clickLoginButton.isDisplayed());
+		clickLoginButton.click();
+	}
+
+	@FindBy(how = How.XPATH, using = "//*[@id='menu.link.customers']/a")
+	private WebElement clickCustomerTab;
+
+	/**
+	 * Method to click on Customer tab after successful login.
+	 * 
+	 * @throws IOException
+	 */
+	public void clickCustomerTab() throws IOException {
+		log.info("Click on Products Tab after successful login");
+		JavaScriptExec.sleep();
+		Assert.assertTrue(clickCustomerTab.isDisplayed());
+		clickCustomerTab.click();
+	}
+	
+		// @FindBy(how=How.XPATH, using="//a[@class='cell double']//*[text()='Customer
+		// A']")
+		private WebElement clickCustomerName;
+
+		/**
+		 * Method to click on customer name.
+		 * 
+		 * @throws IOException
+		 */
+		public void clickCustomerName() throws IOException {
+			CreateOrderPage sp = new CreateOrderPage(driver);
+			log.info("Click on Customer Name.");
+			JavaScriptExec.sleep();
+			String CustomerName = BasePage.getCellData(xlsxName, sheetName, 3, 10);
+			driver.findElement(By.xpath("//*[text()='" + CustomerName + "']/following::a[1]")).click();
+		}
+		
+		// @FindBy(how=How.XPATH, using="//a[@class='cell double']//*[text()='Customer
+				// A']")
+				private WebElement clickCustomerName1;
+
+				/**
+				 * Method to click on customer name.
+				 * 
+				 * @throws IOException
+				 */
+				public void clickCustomerName1() throws IOException {
+					CreateOrderPage sp = new CreateOrderPage(driver);
+					log.info("Click on Customer Name.");
+					JavaScriptExec.sleep();
+					String CustomerName = BasePage.getCellData(xlsxName, sheetName, 3, 4);
+					driver.findElement(By.xpath("//*[text()='" + CustomerName + "']/following::a[1]")).click();
+					navigateBottom();
+				}
+		
+		// @FindBy(how=How.XPATH, using="//*[@id="18306"]/img
+				// A']")
+		//*[@id="18306"]/img
+				private WebElement clickParentCustomerName;
+
+				/**
+				 * Method to click on customer name.
+				 * 
+				 * @throws IOException
+				 */
+				public void clickParentCustomerName() throws IOException {
+					CreateOrderPage sp = new CreateOrderPage(driver);
+					log.info("Click on Customer Name.");
+					JavaScriptExec.sleep();
+					String CustomerName = BasePage.getCellData(xlsxName, sheetName, 3, 5);
+					driver.findElement(By.xpath("//*[text()='" + CustomerName + "']/following::a[1]")).click();
+				}
+	
+	@FindBy(how = How.XPATH, using = "//*[@id='column2']/div[1]/div[7]/div[1]/a[3]/span")
+	//*[@id="column2"]/div[1]/div[7]/div[1]/a[3]/span
+	private WebElement clickEditthisorder;
+
+	/**
+	 * Method to click on Customer tab after successful login.
+	 * 
+	 * @throws IOException
+	 */
+	public void clickEditthisorder() throws IOException {
+		log.info("Click on Edit this folder button");
+		JavaScriptExec.sleep();
+		navigateBottom();
+		Assert.assertTrue(clickEditthisorder.isDisplayed());
```

Comment:
@mayankchhajer When you are using navigateBottom() method then no need to give the sleep() because in the method navigateBottom() sleep is already added there.